themes/default/content/registry/packages/hcp/installation-configuration.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: HCP Setup
-meta_desc: Information on how to install the HCP provider.
+meta_desc: Information on how to install the HashiCorp Cloud Platform Provider for Pulumi.
```

Comment:
The changes to this file are fixes for linting errors that got merged from an unrelated PR: https://github.com/pulumi/registry/pull/854/files.  I just addressed the two errors to unblock my build (meta tag too short and missing blank lint).